tests/libtest.sh

```diff
@@ -80,7 +80,11 @@ fi
 if test -n "${OT_TESTS_VALGRIND:-}"; then
     CMD_PREFIX="env G_SLICE=always-malloc OSTREE_SUPPRESS_SYNCFS=1 valgrind -q --error-exitcode=1 --leak-check=full --num-callers=30 --suppressions=${test_srcdir}/glib.supp --suppressions=${test_srcdir}/ostree.supp"
 else
-    CMD_PREFIX="env LD_PRELOAD=${test_builddir}/libreaddir-rand.so"
+    if test -z "${OT_SKIP_READDIR_RAND:-}"; then
```

Comment:
Would be nice to have a small comment here explaining why this would be necessary (and remind us to investigate in the future).